### PR TITLE
Conformance results for v1.18/k3s

### PR DESCRIPTION
**Pull request by ShylajaDevadiga**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [x] Did you include the product/project logo in SVG, EPS or AI format?  
* [x] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] If your product/project is open source, did you include the `repo_url`?
* [x] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

### CODE REVIEW

**Comment by cjellick** [comment]
k3s has submitted before, but this checklist is new. Do we need to fill out the participation form if we've successfully submitted previous versions?

**Comment by taylorwaggoner** [comment]
No, you don't need a new participation form, so you can go ahead a check the box.  We do ask that you make sure your README meets the requirements for the last bullet - we're looking for detailed steps to ensure that customers could easily recreate your conformance results if desired.  Thanks!

**Comment by cjellick** [comment]
Thanks for the quick response. Will do.

File: v1.18/k3s/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# k3s
+Lightweight Kubernetes
+## To Reproduce
+
+1. Create two machines, any modern Linux will work but this test was done with Ubuntu 20.04
+2. On machine one download k3s v1.18.4+k3s1 and start the server
```
Comment (cjellick) [inline v1.18/k3s/README.md:6]: @ShylajaDevadiga if there is a single install command via curling the install script, perhaps you could include that here

Comment (ShylajaDevadiga) [reply v1.18/k3s/README.md]: Updated README to include curl script.

**Comment by taylorwaggoner** [comment]
You are now Certified Kubernetes

**Comment by cjellick** [comment]
thank you!

**Pull request merged**